src/invidious/videos/parser.cr

```diff
@@ -55,8 +55,9 @@ def extract_video_info(video_id : String, proxy_region : String? = nil)
   client_config = YoutubeAPI::ClientConfig.new(proxy_region: proxy_region)
 
   # Fetch data from the player endpoint
-  # 8AEB param is used to fetch YouTube stories
```

Comment:
We can safely remove the `8AEB` param since stories was entirely removed by YouTube back in late June